sql/2020/22_HTTP_2/percentage_of_resources_loaded_over_HTTP_by_version_per_site.sql

```diff
@@ -0,0 +1,36 @@
+# standardSQL
+# Percentage of resources loaded over HTTP by version per site
+SELECT
+  client,
+  APPROX_QUANTILES((http_0_9/num_requests) * 100, 1000)[OFFSET(500)] AS pct_http_0_9,
```

Comment:
`percentile` is unused

Reply:
`500` was changed to `percentile * 10` on lines 5 - 9